pkg/gui/presentation/commits_test.go

```diff
@@ -72,6 +74,54 @@ func TestGetCommitListDisplayStrings(t *testing.T) {
 		sha2 commit2
 						`),
 		},
+		{
+			testName: "show local branch head",
+			commits: []*models.Commit{
+				{Name: "commit1", Sha: "sha1"},
+				{Name: "commit2", Sha: "sha2"},
+				{Name: "commit3", Sha: "sha3"},
+			},
+			branches: []*models.Branch{
+				{Name: "current-branch", CommitHash: "sha1", Head: true},
+				{Name: "other-branch", CommitHash: "sha2", Head: false},
+				{Name: "master", CommitHash: "sha3", Head: false},
+			},
+			startIdx:                 0,
+			length:                   3,
+			showGraph:                false,
+			bisectInfo:               git_commands.NewNullBisectInfo(),
+			cherryPickedCommitShaSet: set.New[string](),
+			now:                      time.Date(2020, 1, 1, 0, 0, 0, 0, time.UTC),
+			expected: formatExpected(`
+		sha1 commit1
+		sha2 (*) commit2
+		sha3 commit3
+						`),
+		},
+		{
```

Comment:
Can we also add a test for the git bisect case?

Reply:
Now that the bisect logic has moved to `local_commits_context.go`, no we can't. But the `bisect/basic.go` integration test covers this.